benchmarks/asv_delegated_conda.py

```diff
@@ -99,7 +100,7 @@ def name(self):
     def _update_info(self) -> None:
         """Make sure class properties reflect the actual environment being used."""
         # Follow symlink if it has been created.
-        actual_path = Path(self._path).resolve()
+        actual_path = Path(self._path).resolve()  # type: ignore[has-type]
```

Comment:
Changed as suggested. I did not do this initially because I did not want to make assumptions about private attributes of the ancestor class. In general, I would recommend against using such attributes from a child class, as they may change or be removed in any patch release without prior notice, though I admit that in some cases there is no way around it.

In this case, a more reliable solution could be to do something like `conf.env_dir = Path(conf.env_dir).resolve().as_posix()` in `CondaDelegated.__init__` before passing `conf` to the superclass.

Reply:
This whole ASV plugin contains some suspect behaviour. Thank you for the advice, I will see if it is applicable anywhere in #5963